gcloud/storage/acl.py

```diff
@@ -345,78 +346,84 @@ def get_entities(self):
         self._ensure_loaded()
         return list(self.entities.values())
 
-    def reload(self):
-        """Reload the ACL data from Cloud Storage.
+    @property
+    def connection(self):
+        """Compute the connection for API requests for this ACL.
```

Comment:
Why introduce this property if the end goal of #825 is to unbind connection from objects?

Also, docstring still indicates this is a method, but it is a property.